fixtures/fixtures.js

```diff
@@ -56,11 +56,14 @@ var Fixtures = {
     },
     voucherStatus: {
         "success":
-            {"success":["RVNT12345678"], "fail":[], "invalid":[]},
+            {"success":["RVNT12345678"], "fail":[], "invalid":[], "message": 'test'},
```

Comment:
Whoops, I left "test" in from when I was testing the fixture change. Will change to reflect the new response.